crescent/internal/handle_resp.py

```diff
@@ -72,7 +72,19 @@ async def handle_resp(event: InteractionCreateEvent):
                 break
 
     else:
-        await command.callback(ctx, **callback_options)
+        try:
+            await command.callback(ctx, **callback_options)
+        except Exception as e:
+            if hdlrs := command.app._error_handler.registry.get(e.__class__, None):
```

Comment:
`None` is redundant here

Reply:
registry is a normal dict now, not a defaultdict

Reply:
.get() returns None by default for normal dicts

Reply:
Fixed